server/routerlicious/package.json

```diff
@@ -39,12 +40,14 @@
 		"generate:packageList:internal-test": "flub list server --no-private --feed internal-test --outFile feeds/internal-test.txt",
 		"generate:packageList:public": "flub list server --no-private --feed public --outFile feeds/public.txt",
 		"preinstall": "node scripts/only-pnpm.cjs",
-		"layer-check": "cd ../.. && npm run layer-check",
+		"layer-check": "flub check layers --info ../../layerInfo.json",
 		"lint": "npm run prettier && npm run ci:eslint",
 		"lint:fix": "npm run prettier:fix && pnpm run -r --no-sort --stream lint:fix",
-		"policy-check": "cd ../.. && npm run policy-check",
-		"policy-check:asserts": "cd ../.. && npm run policy-check:asserts",
-		"policy-check:fix": "cd ../.. && npm run policy-check:fix",
+		"policy-check": "flub check policy",
```

Comment:
This doesn't work because `check policy` doesn't handle file paths properly and always treats the current dir as the root for the paths.

Reply:
#24249

Reply:
Fix will be in build-tools 0.56.0.